packages/fetchai/skills/tac_control/game.py

```diff
@@ -441,7 +441,7 @@ def __eq__(self, other):
         """Compare to another object."""
         return (
             isinstance(other, Transaction)
-            and super.__eq__()
+            and super().__eq__(other)
```

Comment:
small change needed for `game.py` module.

Reply:
Good!